server/form-pages/apply/reasons-for-placement/basic-information/placementDate.test.ts

```diff
@@ -27,14 +25,14 @@ describe('PlacementDate', () => {
   })
 
   describe('constructor', () => {
-    describe('when the release date is in the future', () => {
+    describe('when the release date is after the application is created', () => {
       it('sets the title and body correctly', () => {
```

Comment:
Suggestion: This is me being new to the code base but what does it meant to be set correctly here? Is it: "it sets the title and body with the release date as the default start date"?

Same comment on the next test if so